src/js/popup.js

```diff
@@ -153,6 +153,19 @@ function loadTemplateEditor (callback = false) {
             callback();
         }
     });
+
+    // Check the "rate" flag. If the flag is not set, display "rate it now" button
+    chrome.runtime.sendMessage({JDTIfunction: 'getRateStatus'}, function (response) {
+        if (response.status === 'success') {
+            if (response.data === true) {
+                // Hide button
```

Comment:
If we don't need to actually hide it (because it's default is hidden) then we should check for the negative and only have the one action. I.e no else clause